src/wallet/rpc/addresses.cpp

```diff
@@ -541,6 +541,10 @@ RPCHelpMan getaddressinfo()
                         {
                             {RPCResult::Type::STR, "label name", "Label name (defaults to \"\")."},
                         }},
+                        {RPCResult::Type::ARR, "use_txids", "",
```

Comment:
Looking at other RPCs I tried to find a similar naming convention for txids.

I found `listreceivedbyaddress` which uses an array called `txids` and `getblock`  that uses `tx` to return transaction ids in their responses.

Not really consistent, but I was wondering if it would make sense to follow the `listreceivedbyaddress` response here and call the attribute `txids` as well instead of `use_txids`?

Reply:
I think `use_txids` is clearer.